smoke-tests/src/test/groovy/io/opentelemetry/smoketest/SpringBootSmokeTest.groovy

```diff
@@ -11,11 +11,13 @@ import io.opentelemetry.proto.collector.trace.v1.ExportTraceServiceRequest
 import java.util.jar.Attributes
 import java.util.jar.JarFile
 import okhttp3.Request
+import spock.lang.IgnoreIf
 import spock.lang.Unroll
 
+@IgnoreIf({ os.windows })
```

Comment:
Should we instead have `WindowsSmokeTest` or something and have `SmokeTest` base class disable itself on windows?

Reply:
`IgnoreIf` does not work on superclass, you have to use it on every concrete class. But yeah, our hierarchy is not convenient for this: we want to run only `AppServerTest` and not other subclasses of `SmokeTest`. That's the only way I found.

Reply:
I didn't look in detail but was hoping something like `assumeThat` could be used in a base class to control the execution. Mostly want to avoid forgetting this on new tests given windows are the odd-one-out. Can you confirm once more that it's not possible?

Reply:
I didn't find a place to put `assumeThat` on class level :(